### ISSUE TITLE
[2.0.x] parameter with value {} is replaced with null

### ISSUE THREAD

**Issue by rastislavpapp** [ops4j/org.ops4j.pax.logging#516]
environment - karaf 4.4.3, pax logging 2.2.0, using slf4j api and log4j2 implementation

problem:
```
log.info("ABC {} DEF", "test{}test");
```
outputs:
```
ABC testnulltest DEF
```

**Comment by grgrzybek** [comment]
Thanks for report - checking.

**Comment by grgrzybek** [comment]
Just a question - how did you obtain the `log` instance? what's the class of this instance?

**Comment by rastislavpapp** [comment]
log instance is obtained via 
```
this.log = org.slf4j.LoggerFactory.getLogger(loggerName);
```
and it is an instance of `org.ops4j.pax.logging.slf4j.Slf4jLogger`

as I've investigated a little further, in the `Slf4jLogger` the message gets formatted correctly, via these calls:
```
 FormattingTuple tuple = MessageFormatter.arrayFormat(format, argArray);
m_delegate.debug(m, tuple.getMessage(), tuple.getThrowable());
```
after this, `tuple.getMessage()` returns "ABC test{}test DEF", but then it is formatted again in `PaxLoggerImpl`:
```
FormattingTriple ft = FormattingTriple.discover(format, false, arg);
doLog(marker.log4j2Marker(), Level.DEBUG, m_fqcn, ft.getMessage(), ft.getThrowable(), ft.getServiceReference(), ft.getArgArray());
```
After this, ft.getMessage() returns "ABC testnulltest DEF", I guess it took the {} as a place to put an exception (which is not present as a last argument, so it is null)

**Comment by grgrzybek** [comment]
@rastislavpapp I've fixed this issue in 2.2.x branch - thank you very much for spotting, because it wass quite serious bug.

If you need a release, just let me know ;)

cc: @jbonofre
